src/main/kotlin/com/dansplugins/factionsystem/listener/PlayerMoveListener.kt

```diff
@@ -69,6 +69,21 @@ class PlayerMoveListener(private val plugin: MedievalFactions) : Listener {
                             )
                             return@Runnable
                         }
+
+                        // prevent autoclaiming if faction land >= max cumulative power
+                        if (plugin.config.getBoolean("factions.limitLand") && claimService.getClaims(playerFaction.id).size + 1 > playerFaction.power) {
```

Comment:
This is already present in the permission check around line 44 - you should just move it out of that block, we don't need to check twice.